src/graphql/resolvers.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   Mutation: Mutations,
 
   Topic: {
-    oracles: ({ address }, data, { db: { Oracles } }) => Oracles.find({ topicAddress: address }),
+    oracles: ({ address }, data, { db: { Oracles } }) => Oracles.cfind({ topicAddress: address }).sort({blockNum: -1}).exec(),
```

Comment:
why do you need to change it to the async find call? does `Oracles.find({ topicAddress: address }).sort({blockNum: -1}).exec()` not work?

Reply:
also, add spacing for the obj with blockNum

Reply:
we are using `nedb-promise` which is a thin wrapper around nedb, so the cursors are mapped to all the cfunctions, so we need to use `cfind` to get the cursor and then set the sorting parameter.

Reply:
But this was working before using `find`. The wrapper allows for using both the sync and async functions.

Reply:
using `find` will directly return a promise, which is not allowed to set the order.  sort is implemented inside Cursor.

generally db.find(query)  = db.cfind(query).exec(), which the cursor returned by cfind can be used to set `sort, skip and limit`

Reply:
gotcha. hm good to know, so `find` which is normally sync turns into async when using the promisified wrapper.